packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -89,17 +109,40 @@ export class RetriableDocumentStorageService implements IDocumentStorageService
                 if (!canRetryOnError(err)) {
                     throw err;
```

Comment:
I feel it would be great to log an error right here if it's not retryable error.

Reply:
yeah.